### PR TITLE
W-14913522 - RPA Manager release notes 2024-01/2.

### PR DESCRIPTION
**Pull request by jhammann72**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 1]
```diff
@@ -2,6 +2,38 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[january-30-2024]]
+== January 30, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+* Automatic RPA Bot update is supported by RPA Bot 1.5.0 and higher. RPA Manager will support the update starting with the next RPA Bot version (greater than 1.5.0).
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
* RPA Manager now supports automatic RPA Bot updates for bots running version 1.5.0 or later.
+
When a new version of RPA Bot is available, RPA Manager shows a new page that enables you to update installed RPA Bots.
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 2]
```diff
@@ -2,6 +2,38 @@
...
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Run configurations can now be edited or deleted on the second and all following pages. | W-14820123
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| You can now edit and delete run configurations that show on the second and all following pages. | W-14820123
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 3]
```diff
@@ -2,6 +2,38 @@
...
+* Automatic RPA Bot update is supported by RPA Bot 1.5.0 and higher. RPA Manager will support the update starting with the next RPA Bot version (greater than 1.5.0).
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| You can now edit and delete run configurations that show on the second and all following pages. | W-14820123
+| The *Deployment Map's* _Test_ tab can now be opened even if the project manager of one of the processes has been deleted. | W-14808512
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| You can now open the _Test_ tab in *Deployment Map* even if the project manager of one of the processes has been deleted. | W-14808512
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 4]
```diff
@@ -2,6 +2,38 @@
...
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| You can now edit and delete run configurations that show on the second and all following pages. | W-14820123
+| You can now open the _Test_ tab in *Deployment Map* even if the project manager of one of the processes has been deleted. | W-14808512
+| *Finance Analysis* no longer shows an error if the process owner of one of the processes has been deleted. | W-14820620
+| *Bot Workflow Library* is now available in Japanese. | W-14836772
+| The tooltips and descriptions on the details panel of the *Bot Workflow Library* will now be shown properly. | W-14852032
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The tooltips and descriptions on the details panel of the *Bot Workflow Library* now show properly. | W-14852032
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 5]
```diff
@@ -2,6 +2,40 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[january-30-2024]]
+== January 30, 2024
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
[[january-31-2024]]
== January 31, 2024
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 6]
```diff
@@ -2,6 +2,40 @@
...
 
+[[january-31-2024]]
+== January 31, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+* RPA Manager now supports automatic RPA Bot updates for bots running version 1.5.0 or later.
++
+When a new version of RPA Bot is available, RPA Manager shows a new page that enables you to update installed RPA Bots.
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: The new *Update* button in the *Bots* view of the *Bot Management* module enables you to select the bots to update to the latest version available. 
```suggestion
When a new version of RPA Bot is available, RPA Manager shows the *Update* button in the *Bots* view of the *Bot Management* module, which enables you to update installed RPA Bots to the latest version.
```

**Review by sebermuley — APPROVED** [review]

**Pull request merged**